setup.py

```diff
@@ -18,10 +18,10 @@
     long_description_content_type="text/markdown",
     classifiers=[
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: 3.6",
         "License :: OSI Approved :: Apache Software License",
     ],
-    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <4',
+    python_requires='>=3.6, <4',
```

Comment:
Since we are removing 3.5, would it make sense to run `pip-compile` again to re-generate the `requirements.txt` and `dev-requirements.txt` files?

Reply:
I agree we should probably update dev-requirements.txt, but we can also remvoe requirements.txt since this is repo is a library. [In the past](https://github.com/NCI-GDC/cdisutils/commit/c2c1e858aafb57369797d981f42446117d5c76c6) we had dev-requirements.txt hold all the dependencies needed to install and run tests.

Reply:
1. removed requirements.txt
2. removed dev-requirements.in
3. add dev section in extras_require in setup.py
4. recompiled dev-requirements.txt